prow/e2e-suite-rbac-no_auth.sh

```diff
@@ -28,7 +28,9 @@ set -u
 # Print commands
 set -x
 
-RBAC_FILE='install/kubernetes/istio-rbac-beta.yaml'
-
 echo 'Running e2e with rbac, no auth Tests'
-./prow/e2e-suite.sh --rbac_path="${RBAC_FILE}" "${@}"
+if [ "$#" != 0 ]; then
```

Comment:
Same question here

Reply:
same answer :)

Reply:
please revert this, it's a different issue